src/ServicePulse.Host/vue/src/components/PageHeader.vue

```diff
@@ -69,6 +69,12 @@ const displayDanger = computed(() => {
               <span class="navbar-label">Events</span>
             </RouterLink>
           </li>
+          <li :class="{ active: isRouteSelected(routeLinks.throughputReport.root) }">
+            <RouterLink :to="routeLinks.throughputReport.root" exact>
```

Comment:
If and when https://github.com/Particular/ServicePulse/pull/1800 is merged, you should pull this out into a dedicated control and add it to the list. Check out the PR for examples on how this was done.